<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="22388" started-at="2018-05-05T14:21:43Z" finished-at="2018-05-05T14:22:06Z">
    <groups>
    </groups>
    <test name="Test1" duration-ms="16020" started-at="2018-05-05T14:21:43Z" finished-at="2018-05-05T14:21:59Z">
      <class name="script.TestScript">
        <test-method status="PASS" signature="preCondition(java.lang.String)[pri:0, instance:script.TestScript@8a8f7c]" name="preCondition" is-config="true" duration-ms="4956" started-at="2018-05-05T14:21:43Z" finished-at="2018-05-05T14:21:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="FAIL" signature="invalidLogin()[pri:0, instance:script.TestScript@8a8f7c]" name="invalidLogin" duration-ms="1313" started-at="2018-05-05T14:21:48Z" finished-at="2018-05-05T14:21:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at script.TestScript.invalidLogin(TestScript.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLogin -->
        <test-method status="PASS" signature="preCondition(java.lang.String)[pri:0, instance:script.TestScript@8a8f7c]" name="preCondition" is-config="true" duration-ms="5817" started-at="2018-05-05T14:21:50Z" finished-at="2018-05-05T14:21:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="PASS" signature="validLogin()[pri:0, instance:script.TestScript@8a8f7c]" name="validLogin" duration-ms="3259" started-at="2018-05-05T14:21:56Z" finished-at="2018-05-05T14:21:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogin -->
      </class> <!-- script.TestScript -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="22388" started-at="2018-05-05T14:21:43Z" finished-at="2018-05-05T14:22:06Z">
      <class name="script.TestScript">
        <test-method status="PASS" signature="preCondition(java.lang.String)[pri:0, instance:script.TestScript@10b40af]" name="preCondition" is-config="true" duration-ms="11051" started-at="2018-05-05T14:21:43Z" finished-at="2018-05-05T14:21:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="FAIL" signature="invalidLogin()[pri:0, instance:script.TestScript@10b40af]" name="invalidLogin" duration-ms="1160" started-at="2018-05-05T14:21:54Z" finished-at="2018-05-05T14:21:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at script.TestScript.invalidLogin(TestScript.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLogin -->
        <test-method status="PASS" signature="preCondition(java.lang.String)[pri:0, instance:script.TestScript@10b40af]" name="preCondition" is-config="true" duration-ms="5614" started-at="2018-05-05T14:21:56Z" finished-at="2018-05-05T14:22:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="PASS" signature="validLogin()[pri:0, instance:script.TestScript@10b40af]" name="validLogin" duration-ms="4430" started-at="2018-05-05T14:22:01Z" finished-at="2018-05-05T14:22:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogin -->
      </class> <!-- script.TestScript -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
